feeder/src/vote.ts

```diff
@@ -315,13 +318,51 @@ export async function vote(args: VoteArgs): Promise<void> {
         console.error(err.message)
       }
 
+      console.log('vote: lcd client unavailable, rotating to next lcd client.')
+      rotateLCD(client, currentLCDIndex, maxLCDIndex, args)
       resetPrevote()
     })
 
     await Bluebird.delay(Math.max(500, 500 - (Date.now() - startTime)))
   }
 }
 
+function rotateLCD(
+  client: LCDClient,
+  currentLCDIndex: number,
+  maxLCDIndex: number,
+  args: VoteArgs
+) {
+  if (++currentLCDIndex > maxLCDIndex) {
+    currentLCDIndex = 0
+  }
+  client.config.URL = args.lcdAddress[currentLCDIndex]
+  console.log(
+    'Switched to LCD address ' + currentLCDIndex + '(' + args.lcdAddress[currentLCDIndex] + ')'
+  )
+
+  return
+}
+
+async function voteWithLCD(
```

Comment:
Seems no use case here, can you remove it?

Reply:
Done 👍🏼